orttraining/orttraining/test/python/_test_helpers.py

```diff
@@ -9,12 +9,20 @@
 try:
     from onnxruntime.training.ortmodule import ORTModule
     from onnxruntime.training.ortmodule._graph_execution_manager_factory import GraphExecutionManagerFactory
+    from onnxruntime.training.ortmodule._fallback import ORTModuleInitException
 except ImportError:
     # Some pipelines do not contain ORTModule
     pass
-except EnvironmentError:
-    # Some pipelines do not contain ORTModule
-    pass
+except Exception as e:
+    try:
+        from onnxruntime.training.ortmodule._fallback import ORTModuleInitException
+        if isinstance(e, ORTModuleInitException):
+            # ORTModule is present but not ready to run
+            # That is OK because this file is also used by ORTTrainer tests
+            pass
+    except Exception:
+        # ORTModule not present
+        pass
```

Comment:
again, this code seems to be doing nothing.

Reply:
fixed